
export const defaultErrorTitle = 'Ha ocurrido un error';

export const defaultMsg = {
    editedSuccess: {
        title: 'Changes saved',
        msg: 'All changes have been saved successfully'
    },
    editedError: {
        title: defaultErrorTitle,
        msg: 'There was an error trying to save changes'
    }
}

export const projectMsg = {
    createdSuccess: {
        title: 'Project created',
        msg: 'Your project has been created'
    },
    createdError: {
        title: defaultErrorTitle,
        msg: 'There was an error trying to create your project'
    },

    deletedSuccess: {
        title: 'Project deleted',
        msg: 'Your project has been deleted'
    },
    deletedError: {
        title: defaultErrorTitle,
        msg: 'There was an error trying to delete your project'
    }
}

export const taskMsg = {
    createdSuccess: {
        title: 'Task created',
        msg: 'Task created successfully'
    },
    createdError: {
        title: defaultErrorTitle,
        msg: 'There was an error trying to create the task'
    },

    deletedSuccess: {
        title: 'Task deleted',
        msg: 'Task deleted successfully'
    },
    deletedError: {
        title: defaultErrorTitle,
        msg: 'There was an error trying to delete the task'
    },
    completeError: {
        title: defaultErrorTitle,
        msg: 'Task could not be completed'
    }
}



export const collabMsg = {
    addSuccess: {
        title: 'Collaborator added'
    },
    addError: {
        title: defaultErrorTitle,
        msg: 'Collaborator could not be added'
    },
    deletedSuccess: {
        title: 'Collaborator deleted',
        msg: 'Collaborator has been successfully removed'
    },
    deletedError: {
        title: defaultErrorTitle,
        msg: 'Collaborator could not be removed'
    }
}

export const favoriteMsg = {
    Error: {
        title: defaultErrorTitle,
        msg: 'The operation could not be completed'
    }
}



